pyproject.toml

```diff
@@ -33,7 +33,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.7.0"
 typing-extensions = {version = "^4.0.1", python = "<3.8"}
-pytz = "^2021.3"
+pytz = "*"
```

Comment:
I don't think we can relax it this much - very old versions of pytz will presumably have a different API.

If we really want to relax it a lot we should test against the oldest version we allow.